src/converter/bib/BibConversion.js

```diff
@@ -40,6 +131,8 @@ function _convertJournalArticleFromCSLJSON(source) {
   })
 
   if(!data.doi) {
+    // TODO: We should not rely that the imported item has a DOI, because it can also be imported from a generic CSL JSON file.
+    //  However, there are some problems in the further processing withouth a DOI at the moment...
     throw new Error(`Citation must have DOI.`)
```

Comment:
@michael Can you help for entries from CSL JSON without a DOI?

Reply:
This is good for now, but yes we need to think of a solution for this soon. The problem is, that we don't have an easy way to detect duplicates if there's no doi. We'd need to compare if all properties are the same to skip an entry.

Reply:
Okay. Yeah, the ISBN is for example not always suitable as an unique identifier (publisher reuses old ISBN for newer editions etc.).

Reply:
Created a separate issue, which we can tackle when there's time and we've chosen what strategy we will apply: https://github.com/substance/texture/issues/483